packages/apps/docs/src/components/Markdown/Heading/Heading.tsx

```diff
@@ -93,12 +44,54 @@ export const TaggedHeading: FC<IProp> = ({
 
   const slug = createSlug(slugInputStr, index, parentTitle);
 
-  return (
-    <StyledHeader as={as} variant={variant ?? as}>
-      {children}
-      <StyledLinkIcon id={slug} href={`#${slug}`}>
-        <SystemIcons.Link />
-      </StyledLinkIcon>
-    </StyledHeader>
+  const ArticleLink = (
```

Comment:
Could you make this `content` and also put children here? Just to reduce duplicated code